Formula/gtk+3.rb

```diff
@@ -45,6 +45,8 @@ class Gtkx3 < Formula
     depends_on "libxkbcommon"
     depends_on "wayland-protocols"
     depends_on "xorgproto"
+
+    patch :DATA
```

Comment:
```suggestion
    # remove in next release
    patch do
      url "https://gitlab.gnome.org/GNOME/gtk/-/commit/66a199806ceb3daa5e2c7d3a5b45a86007cec46a.patch"
      sha256 "426e0e542b152c50d5cf81ee4710572ee48b98880d1eec4ff12a8c1abe4a079b"
    end
```

Reply:
The full patch adds/changes a lot of stuff. Have you tried to build it on Ubuntu?

Reply:
Maybe add a code commit reference for that (it would be removed in the next release though)

Reply:
@chenrui333 
The above change doesn't work on Ubuntu. 
I looked around and it seems that the :DATA patch should be put into https://github.com/Homebrew/formula-patches/tree/master/gtk%2B3,  mentioning the original GTK's commit and referenced here with  "Remove in next release".

Reply:
it should be the same thing, putting into the formula-patches or adding directly into the formula.

Reply:
Yeah, I will add that commit reference as code comment after this PR

Reply:
Followup PR #117700